app/views/wizard/steps/duty/calculations/_trade_details.html.erb

```diff
@@ -0,0 +1,46 @@
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+    About this commodity code
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key govuk-body-s">
+          Origin:
+        </dt>
+        <dd class="govuk-summary-list__value govuk-body-s">
+          <%= user_session.country_of_origin %>
```

Comment:
We need to extract the real country name here. Maybe for now we just fetch that country, and then later we add it to the session as we discussed?